apps/omg_watcher/config/dev.exs

```diff
@@ -1,11 +1 @@
 use Mix.Config
-
```

Comment:
not 100% sure, but it seems there should be no reason to remove any of these lines. (same for the nearby `prod.exs`)